airflow/www_rbac/static/webpack.config.js

```diff
@@ -0,0 +1,128 @@
+/**
```

Comment:
Does this file actually need to be in the `static/` folder -- is it needed when the webserver is running, or just in the JS built stage?

Reply:
This file needs to be in the same folder as the package.json (A configuration file for npm comamnd)
It is needed when you run any of the `npm run prod|build|dev|etc...` commands. (without running these commands you won't get required JS, CSS). I'll add a detailed comment as how the npm and webpack works.